risc0/zkvm/lib.rs

```diff
@@ -22,24 +24,27 @@ mod ffi {
 
         type MethodID;
 
-        fn new_method_id(elf_path: &CxxString) -> Result<UniquePtr<MethodID>>;
-        fn write(&self, filename: &CxxString) -> Result<()>;
+        fn make_method_id(elf_path: &CxxString) -> Result<UniquePtr<MethodID>>;
     }
 }
 
+// Can we remove this?
```

Comment:
I don't think so?

Reply:
I suppose I should've phrased it "can we reduce the surface area of this api". I don't think it's particularly important but I find those sorts of small challenges useful for building my understanding.